script.js

```diff
@@ -1,69 +1,63 @@
 const cardContainer = document.querySelector(".card-container");
-const newGameButton = document.getElementById("new-game-btn");
+const alertContainer = document.querySelector("[data-id='alert-container']");
+const alertOverlay = document.querySelector("[data-id='alert-overlay']");
```

Comment:
You have very good naming naturally, which is rarely the case for beginners. Very impressive!